custom_components/opnsense/sensor.py

```
@@ -521,40 +568,29 @@
-    def native_value(self):
-        interface = self._opnsense_get_interface()
-        prop_name: str = self._opnsense_get_interface_property_name()
-        try:
-            return interface[prop_name]
-        except (KeyError, TypeError):
-            return STATE_UNKNOWN
-
 
 class OPNsenseCarpInterfaceSensor(OPNsenseSensor):
-    def _opnsense_get_interface_name(self):
+    def _opnsense_get_carp_interface_name(self):
```

Ruff

Missing return type annotation for private function `_opnsense_get_carp_interface_name`

(ANN202)